packages/date-time/src/components/CalendarDayGrid/CalendarDayGrid.styles.ts

```diff
@@ -104,12 +105,46 @@ export const styles = (props: ICalendarDayGridStyleProps): ICalendarDayGridStyle
       fontWeight: FontWeights.regular,
       color: palette.neutralPrimary,
       cursor: 'pointer',
+      position: 'relative',
       selectors: {
+        [HighContrastSelector]: {
+          color: 'WindowText',
+          backgroundColor: 'Window',
+          zIndex: 0,
+        },
         ['&.' + classNames.hoverStyle]: {
           backgroundColor: palette.neutralLighter,
+          selectors: {
+            [HighContrastSelector]: {
+              zIndex: 3,
+              overflow: 'visible',
+              color: 'WindowText',
+              backgroundColor: 'Window',
```

Comment:
do you need these 2 lines or we can relay on cascaded styles from above

Reply:
can remove color, backgroundColor gets overridden right above this so need to re-override